test/e2e_node/e2e_node_suite_test.go

```diff
@@ -54,6 +54,9 @@ import (
 	"github.com/onsi/gomega"
 	"github.com/spf13/pflag"
 	"k8s.io/klog/v2"
+
+	// test sources
+	_ "k8s.io/kubernetes/test/e2e_node/gcp"
```

Comment:
?

Reply:
I gather this should have been here before and wasn't, don't see a move to this dir as part of this PR

Reply:
Yes - https://github.com/kubernetes/kubernetes/pull/99319 effectively broke it.

Reply:
I replaced with the imports pattern here too.